components/WelcomeMessage/index.js

```diff
@@ -0,0 +1,92 @@
+import Heading from "../Heading";
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+
+export const Welcome = styled.text`
```

Comment:
```suggestion
export const Welcome = styled.p`
```
Text is not a valid html element